frontend/src/constants/liveTail.ts

```diff
@@ -0,0 +1,3 @@
+export const LIVE_TAIL_HEARTBEAT_TIMEOUT = 10 * 60 * 1000;
```

Comment:
Comment the logic around LIVE_TAIL_HEARTBEAT_TIMEOUT .

Can directly use `600000` in place of `10 * 60 * 1000`.